packages/bootstrap-vue-next/src/utils/dom.ts

```diff
@@ -230,3 +230,21 @@ export const getTransitionDelay = (element: HTMLElement) => {
   const transitionDurationMs = Number(transitionDuration.slice(0, -1)) * 1000
   return transitionDelayMs + transitionDurationMs
 }
+
+/**
+ * convert unknown input to HTMLElement
+ *
+ * @param target
+ */
+export const getElement = (
+  target: MaybeRefOrGetter<string | ComponentPublicInstance | HTMLElement | null>
+): HTMLElement | undefined => {
+  const element = toValue(target)
```

Comment:
```suggestion
  target: string | ComponentPublicInstance | HTMLElement | null
): HTMLElement | undefined => {
  const element = target
```
'element' likely doesn't need to exist anymore

The consumption of the util should pass in `.value`, not the Ref

Reply:
so it should be in composables, or the toValue should be on calling end? Which do we want to prefer?

Reply:
The value should be given. So, getElement(el.value)

Reply:
so getElement(toValue(el)) since we don't check what user has given us?

Reply:
toValue would be if what you have is a MaybeRefOrGetter (could be 1, could be Ref(1) or could be () => 1)

So if you have any of those, and dont know which one it is, then you'd use toValue

If you have a Ref, you'll want to use .value, as its just a ref

So, it depends on what you're working with

Reply:
So if I want to keep the toValue there because I don't care what user inputs and I don't want to remember to check  on multiple places on the caller end, it's a composable? For me the better DX is to have the function check it self, so dev doesn't have to care.

Reply:
The issue to me really comes down to the single responsibility principle in that it should only be getting the value, not converting the parameter into the desired format, then getting the value

Reply:
This has nothing to do with scrollspy so I returned the function to BPopover.vue and copypasted it to useScrollspy.ts